WcaOnRails/app/views/polls/preview.html.erb

```diff
@@ -0,0 +1,23 @@
+<% provide(:title, "Preview poll ##{@poll.id}") %>
+
+<div class="container">
```

Comment:
I worry it will be hard to keep this file in sync with our `vote.html.erb` file. If someone adds a new field, I don't think they'll remember to change this file. Can we combine this with `votes.html.erb`?